ci/common/test-pyclaragenomics.sh

```diff
@@ -14,7 +14,16 @@
 ######################################
 set -e
 
-cd $1
+PYCLARAGENOMICS_DIR=$1
+cd $PYCLARAGENOMICS_DIR
+
+#Install external dependencies.
 python3 -m pip install -r requirements.txt
-python3 setup.py install
-python3 -m pytest -s
+python3 setup.py develop
```

Comment:
@vellamike doesn't work when I do `python3 setup.py install` but works with `develop`. Will need to figure that out.